app/jinja_filters.py

```diff
@@ -7,7 +7,7 @@
 import flask_babel
```

Comment:
There is some duplication between this file and `placeholder_transforms.py`. This file should just contain filters that are called from templates. Longer term we will remove most of the filters and apply the transforms before the values get rendered in a template.

Reply:
Yeh I forgot to go through and remove the old ones